script.js

```diff
@@ -0,0 +1,144 @@
+// funtion reacts to button
+
+const myFunc = () => {
+
+    cityApi = document.getElementById('europe').value
+    const today = document.getElementById('currentWeather')
+    const orginalApi = 'https://api.openweathermap.org/data/2.5/weather?q=London&appid=2b9468766d0e54560c7e599762d2e80b'
+    const weatherapi = 'https://api.openweathermap.org/data/2.5/weather?q='
+    const apiId = '&appid=2b9468766d0e54560c7e599762d2e80b'
+    const newApi = `https://api.openweathermap.org/data/2.5/weather?q=${cityApi}&appid=2b9468766d0e54560c7e599762d2e80b`
+
+    //fetch today
+    fetch(newApi).then((response) => {
+        return response.json()
+    }).then((json) => {
+        // dealing with json
+        const times = (new Date(json.dt * 1000)).toUTCString().split(' ') // dealing with date
+        const suny = [json.sys.sunrise, json.sys.sunset]
+        console.log("sun", suny)
+        const sunyNew = suny.map((item) => {
+            return (new Date(item * 1000)).toUTCString().split(' ')
+        })
+        const sunyList = sunyNew.map((item) => {
+            const clock = item[4].split(':')
+            return `${clock[0]}:${clock[1]}`  // removing seconds
+        })
+        const noSecond = times[4].split(':')
+        // my weather object contains and sort relevant information
+        let weatherObject = {
+            city: json.name,
+            name: json.weather[0].main,
+            description: json.weather[0].description,
+            temp: `${(json.main.temp - 273.15).toFixed(1)}&#8451;`,
+            weekDay: times[0].replace(',', ''),
+            date: `${times[1]} ${times[2]} ${times[3]}`,
+            time: `${noSecond[0]}:${noSecond[1]}`,
+            rise: sunyList[0],
+            set: sunyList[1]
+        }
+
+
+        // changing color, text, and image based on weather
+        const icons = {
+            clouds: { image: "media/clouds.svg", bgColor: '#F4F7F8', fontColor: '#F47775', head: `Light a fire and get cozy. ${weatherObject.city} is looking gray today` },
+            rain: { image: "media/rain.svg", bgColor: '#A3DEF7', fontColor: '#164A68', head: `Don't forget your umbrella Its wet in ${weatherObject.city} today`, },
+            clear: { image: "media/clear.svg", bgColor: '#F7E9B9', fontColor: '#2A5510', head: `Get your sunnies on. ${weatherObject.city} is looking rather great today` }
+        }
+
+        const pictures = () => {
+            console.log("inside function", weatherObject.name)
+            let choice = "blank"
+            if (weatherObject.name === "Clouds" || weatherObject.name === "Mist") {
+                choice = icons.clouds
+            }
+            else if (weatherObject.name === "Rain" || weatherObject.name === "Snow") {
+                choice = icons.rain
+            }
+            else if (weatherObject.name === "Clear") {
```

Comment:
Else will be enough here!
```suggestion
            else (weatherObject.name === "Clear") {
```